tests/unit/components/ParameterForm.spec.ts

```diff
@@ -133,12 +150,16 @@ describe("parameter form", () => {
     expect(rangeInput.element.value).toBe("2000");
 
     await shortList.find("input[value='yes']").setChecked();
-    expect(numericInput.element.value).toBe("17000");
-    expect(rangeInput.element.value).toBe("17000");
+    setTimeout(() => {
+      expect(numericInput.element.value).toBe("17000");
+      expect(rangeInput.element.value).toBe("17000");
+    }, 500);
 
     await shortList.find("input[value='no']").setChecked();
-    expect(numericInput.element.value).toBe("2000");
-    expect(rangeInput.element.value).toBe("2000");
+    setTimeout(() => {
+      expect(numericInput.element.value).toBe("2000");
+      expect(rangeInput.element.value).toBe("2000");
+    }, 500);
```

Comment:
I'm surprised you need these `setTimeouts` - nextTick should do the trick? I'm also a bit confused about what shortList represents here - this is a use case we don't currently have where button group param potentially has dependent params to update?

Reply:
nextTick works great, somehow I forgot about it.